optaplanner-core/src/main/java/org/optaplanner/core/api/score/holder/ScoreHolder.java

```diff
@@ -92,7 +100,9 @@
      * @return never null
      * @throws IllegalStateException if {@link #isConstraintMatchEnabled()} is false
      * @see ScoreDirector#getConstraintMatchTotalMap()
+     * @deprecated for removal from public API
```

Comment:
Let's reference ScoreManager because if they're using this, they probably want to learn about ScoreManager.